src/reducers/updateThoughts.ts

```diff
@@ -122,8 +121,20 @@ const updateThoughts = (state: State, { thoughtIndexUpdates, contextIndexUpdates
     state => ({
       ...state,
       expanded: expandThoughts(state, state.cursor),
-    })
+    }),
+
+    !state.cursorInitialized ? state => {
+      const { path, contextViews } = decodeThoughtsUrl(state, window.location.pathname)
+      return !path || path.some(pathItem => pathItem.id === '') ?
```

Comment:
The empty id seems like a tenuous thing to code against. That is, I'm skeptical about treating that as a formal API. I'd like to consider other ways to detect this.

Reply:
You are right. I didn't like it.  I updated the code, now we are checking whether thought is loaded using thoughtIndex.